openapi.yaml

```diff
@@ -221,11 +229,13 @@ paths:
 components:
   schemas:
     TicketType:
-      description: Type of ticket being purchased. Use `general` for regular museum entry and `event` for tickets to special events.
+      description: Type of ticket being purchased. Use `general` for regular museum entry and `event` for tickets to special events. Use `student` for school-aged children. Visitors aged 0-5 are free.
```

Comment:
For clarity, could we rephrase this to put the general, student and child tickets in one sentence, and the special event ones in another? I think it's age-related general entry, or an event ticket.